Mathlib/Combinatorics/Enumerative/DoubleCounting.lean

```diff
@@ -71,6 +72,12 @@ theorem sum_card_bipartiteAbove_eq_sum_card_bipartiteBelow [∀ a b, Decidable (
   simp_rw [card_eq_sum_ones, bipartiteAbove, bipartiteBelow, sum_filter]
   exact sum_comm
 
+theorem sum_sum_bipartiteAbove_eq_sum_sum_bipartiteBelow
```

Comment:
This lemma doesn't need to be stated in terms of the real numbers - and doing so adds a lot of imports to this file unnecessarily. I believe all you really need is an `AddCommMonoid`.